src/events/readySweeper.ts

```diff
@@ -6,7 +6,7 @@ const HEADER = `\u001B[39m\u001B[94m[MEMORY CLEANUP]\u001B[39m\u001B[90m`;
 export default class extends Event {
 
 	public constructor(store: EventStore, file: string[], directory: string) {
-		super(store, file, directory, { event: 'ready' });
```

Comment:
I... don't think that runs only once

Reply:
It's for the `ready` event, which is only fired once

Reply:
Ready is only triggered once, specifically, when the bot is ready. It is triggered immediately before KlasaReady.

Reply:
whoops, didn't see that

/resolved